src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -29,6 +29,7 @@ class Yaml
     const DUMP_OBJECT_AS_MAP = 64;
     const DUMP_MULTI_LINE_LITERAL_BLOCK = 128;
     const PARSE_CONSTANT = 256;
+    const DUMP_EMPTY_ARRAY_AS_SEQUENCE = 512;
```

Comment:
Sorry for being late here, but I had no time during the weekend to do the final review. This value must be adjusted as 512 is already used by the new tags support feature (see #21678).